src/main/java/kitEditor/KitEditor.java

```diff
@@ -897,6 +905,7 @@ private void addSample(File wavFile) {
     }
 
     private void renameSample(int sampleIndex, String sampleName) {
+        samples[selectedBank][sampleIndex].setName(sampleName);
```

Comment:
what is the reason for this change?

Reply:
`Sample.name` needs to be changed when renaming the sample if a new duplicate sample should retain the name when the original has been previously renamed.